python/lsst/obs/hsc/hscMapper.py

```diff
@@ -192,7 +192,7 @@ def std_raw(self, item, dataId):
 
     def std_dark(self, item, dataId):
         exposure = self._standardizeExposure(self.calibrations['dark'], item, dataId, trimmed=False)
-        visitInfo = afwImage.makeVisitInfo(exposureTime=1.0)
+        visitInfo = afwImage.makeVisitInfo(exposureTime=1.0, darkTime=1.0)
```

Comment:
The dark construction script has never set the dark time.  I'll address that in [DM-8913](https://jira.lsstcorp.org/browse/DM-8913).